JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -39,6 +39,14 @@
     createLazyMethod("setAuthenticatedUserContext");
     createLazyMethod("clearAuthenticatedUserContext");
 
+    createLazyMethod("startTrackEvent");
+    createLazyMethod("stopTrackEvent");
+
+    createLazyMethod("startTrackPage");
+    createLazyMethod("stopTrackPage");
+
+    createLazyMethod("flush");
```

Comment:
Assigning duplicated strings to local variables allows them to be minified to safe space. 
This will reduce the snippet size by 13 bytes.

    const userContext = "AuthenticatedUserContext";
    const trackEvent = "TrackEvent";
    const trackPage = "TrackPage";
    const start = "start";
    const stop = "stop";

    createLazyMethod("set" + userContext);
    createLazyMethod("clear" + userContext);

    createLazyMethod(start + trackEvent);
    createLazyMethod(stop + trackEvent);
    
    createLazyMethod(start + trackPage);
    createLazyMethod(stop + trackPage);